bld/android/project.properties

```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-14
+target=android-21
```

Comment:
Looks like `log2` wasn't added until version 18 for arm/x86 and 21 for arm64/x64.

Reply:
IMHO, this should be an acceptable requirement bump.

Framework | Min Android
--- | ---
MAUI | 21
MAUI Blazor | 24
Unity | 22